src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function summator(...arr) {
```

Comment:
The 'summator' function uses rest parameters to collect arguments, which is good for flexibility, but it does not match the pattern described in the task. The function should accept a single argument at a time, not an array of arguments.